props/blaster.h

```diff
@@ -152,51 +204,42 @@ class Blaster : public PropBase {
     PollNextAction();
   }
 
-  // Make clash do nothing
-  void Clash(bool stab) override {}
+  // Make clash do nothing except unjam if jammed.
+  void Clash(bool stab) override {
+    if( is_jammed_ ) {
```

Comment:
if (is_jammied_) {

Reply:
Ah thanks. Yeah my IDE code autostyler sometimes tries to be too helpful.

Reply:
Fixed.